HS-302: Update README (#270)
OpenNMS-Cloud/horizon-stream: README.md

> # Horizon Stream 🚀

Reviewer: Need to determine if stream is the name we are going to use even at this early stage. That is a marketing call.

> Horizon Stream is the future of OpenNMS Horizon, an open source network monitoring.

Reviewer: ```suggestion
Horizon Stream is the future of OpenNMS Horizon, an open source network monitoring platform.
```

> It aims to be an easy to use, secure, reliable and extensible open-source network monitoring plattform.

Reviewer: These two lines require a bit of messaging. They imply that Horizon doesn't already give them this. How can we position Stream effectively?

The goal is to give our users network visibility and actionable insights helping them to provide better IT services. 

> ☣️ The project is in a very early stage and things break and move around. It is not ready to run production workloads.

Reviewer: ```suggestion
☣️ The project is in a very early stage and things break and move around.
It is not ready to run production workloads.
```

## Supported Platforms

To deploy the whole application a [Kubernetes (K8s)](https://kubernetes.io/) cluster is required.

> As it is right now you need a K8s cluster with 24GB RAM.

Reviewer: ```suggestion
Currently you need a K8s cluster with 24GB RAM.
```

## Documentation

> We use the GitHub Wiki in the Horizon Stream GitHub project for documentation of the project.

Reviewer: ```suggestion
We use the GitHub Wiki in the Horizon Stream GitHub project to document the project.
```

It gives us enough flexibility and structure to migrate it later to a more sophisticated versioned documentation framework.

> You find the documentation how to get it installed in the [administrators getting started](https://github.com/OpenNMS/horizon-stream/wiki/Getting-Started---Admin) section.

Reviewer: ```suggestion
The following sections might be of most interest: 
* Installation in the [administrators getting started](https://github.com/OpenNMS/horizon-stream/wiki/Getting-Started---Admin) section.
```

> If you are interested in building it from source you find instruction in our [developers getting started](https://github.com/OpenNMS/horizon-stream/wiki/Getting-Started) chapter.

Reviewer: ```suggestion
* Building from source  in the [developers getting started](https://github.com/OpenNMS/horizon-stream/wiki/Getting-Started) section.
```

> In case you want to contribute code you can find instructions in our [contribution guidelines](https://github.com/OpenNMS/horizon-stream/wiki/Development-Guidelines).  

Reviewer: ```suggestion
* [Contribution guidelines](https://github.com/OpenNMS/horizon-stream/wiki/Development-Guidelines), if you want to contribute code.
```

> In case you run into problems, you can look into [troubleshooting](https://github.com/OpenNMS/horizon-stream/wiki/Troubleshooting) for quick help.

Reviewer: ```suggestion
* [Troubleshooting](https://github.com/OpenNMS/horizon-stream/wiki/Troubleshooting) for quick help.
```

If you have questions feel free and talk to people in our community in our [opennms-discussion](https://chat.opennms.com/opennms-discuss) channel in Mattermost.

## Questions

### How does this relate to OpenNMS Horizon?

> To get these principles addressed without breaking people's current solutions and implementations.

Reviewer: What principles are we addressing?

> We know people in the OpenNMS community and commercial customers spent time and money to integrate OpenNMS Horizon into their environments. OpenNMS Horizon gets the maintenance and enhancements through the OpenNMS Plugin API extensions without a change.

Reviewer: ```suggestion
We know people in the OpenNMS community and commercial customers spent time and money to integrate OpenNMS Horizon into their environments.
OpenNMS Horizon gets the maintenance and enhancements through the OpenNMS Plugin API extensions without a change.
```

Reviewer: Also, what maintenance and enhancements? From Stream?

> We can't do a major re-architecture of the platform without breaking things for our current users.

Reviewer: It is not clear from earlier that we are doing a re-architecture of the platform. Suggest being clear about that near the beginning.

> Our goal is to take over the OpenNMS Plugin API as an integration contract, and we aim to make the existing plugins compatible.

Reviewer: ```suggestion
Our goal is to take over the OpenNMS Plugin API as an integration contract, and we aim to make the existing plugins compatible.
```
Integration with Horizon?
